src/isPalindrome.js

```diff
@@ -0,0 +1,14 @@
+export default function isPalindrome(str) {
+  const formattedStr = str.toLowerCase().replace(/[^a-z|0-9]/g, "")
+  function reversedString(formattedStr) {
+
+    if (formattedStr === ""){
+      return "";
+    } else {
+      return reversedString(formattedStr.substr(1)) + formattedStr.charAt(0)
+    }
+  }
+  return formattedStr == reversedString(formattedStr)
```

Comment:
i think you can just do 
``` return formattedStr == reversedString(formattedStr) ```
and it'll evaluate its truthiness without the ternary